git-gerrit.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export LANG='en_US.UTF-8'
+
+python3 -B ./git-gerrit
```

Comment:
python2 is not dead (yet)

Reply:
I've added some dependency checking logic.